code/datums/black_market_item.dm

```diff
@@ -280,6 +280,39 @@ for anyone but the person committing mass murder.
 	cost_max = 600
 	display_chance = 80
 
+/datum/black_market_item/animal/monkeyhide
+	name = "Monkey Hide"
+	desc = "You're own peice of leather, some assembly required."
```

Comment:
>You're own peice

if these are ironic typos I fail to see the humour

Reply:
It isn't, working on it now

Reply:
YOU FUCKED UP BATHOS.